src/mainboard/clevo/adl-p/variants/ns70pu/gpio.c

```diff
@@ -0,0 +1,227 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
Why not dump GPIOs using inteltool we've just developed, and use intelp2c to generate this file?

Reply:
Will be done later